web-components/src/components/icons/PauseIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
can you add this to icon.stories.tsx?

Reply:
sorry if I haven't mentioned that before to you, but it's best to build new icons without using material ui in case we move away from it at some point, see for instance `OtherDocumentsIcon`, any custom style can just be passed through `className` and then we use "currentColor" in the svg code so that we can customize the icon color, just by changing the text color in the className prop